erts/emulator/beam/erl_process.h

```diff
@@ -673,6 +673,7 @@ struct ErtsSchedulerData_ {
     } pending_signal;
 
     Uint64 reductions;
+    Uint64 rand_state;
```

Comment:
This would be super-amazing to have for @RaimoNiskanen RNG improvements. It would be possible to reuse this per-scheduler state for mwc59.
Right now the problem with mwc59 is lack of initial state (seed needs to be generated and kept on the process heap). But if we have this per-scheduler random state already in, we can do `mwc59` with no extra state needed.